qsignature/src/org/qcmg/sig/QSigCompareDistance.java

```diff
@@ -45,6 +45,11 @@
 
 /**
  * The Class QSigCompareDistance.
+ * 
+ * This uses the Euclidean distance method to compare positions.
+ * This class has been superseded by the Compare class and so is deprecated. 
+ * 
+ * @deprecated
```

Comment:
please move  "deprecated" annotation begin of the class, as this class should be deprecated or delete

Reply:
done